examples/internal/integration/integration_test.go

```diff
@@ -1574,51 +1632,102 @@ func testResponseStrings(t *testing.T, port int) {
 		t.Fatalf("waitForGateway(ctx, %d) failed with %v; want success", port, err)
 	}
 
-	for i, spec := range []struct {
-		endpoint     string
-		expectedCode int
-		expectedBody string
-	}{
-		{
-			endpoint:     fmt.Sprintf("http://localhost:%d/responsestrings/foo", port),
-			expectedCode: http.StatusOK,
-			expectedBody: `["hello","foo"]`,
```

Comment:
Tests that relied on exact JSON formatting have had to be rewritten to marshal to types before comparing (see https://github.com/golang/protobuf/issues/1121)